### PR TITLE
Add a Zip Classes

### PR DESCRIPTION
**Pull request by FerHinojosa**
Add zip classes

### CODE REVIEW

File: src/main/java/com/jalasoft/webservice/controller/FileManager.java [hunk 1]
```diff
@@ -29,6 +29,7 @@ public static String getFilePath(@RequestParam("file") MultipartFile file) throw
         Utils utils = new Utils();
         String filePath = utils.getTemp() + file.getOriginalFilename();
         Path location = Paths.get(filePath);
+        System.out.println(filePath);
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/controller/FileManager.java]: System.out.println must not be in our code.

File: src/main/java/com/jalasoft/webservice/model/ImageConvert.java [hunk 2]
```diff
@@ -35,7 +35,8 @@ public Response convert(Criteria criteria) throws IOException {
 
         try {
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/model/ImageConvert.java]: Normally there is not a space line after a statement.

File: src/main/java/com/jalasoft/webservice/model/ImageConvert.java [hunk 3]
```diff
@@ -47,14 +48,19 @@ public Response convert(Criteria criteria) throws IOException {
             //Instantiating the PDFRenderer class
             PDFRenderer renderer = new PDFRenderer(document);
             int count = document.getNumberOfPages();
+            String [] filePaths = new String[count];
+
             for (int page = 0; page < count; ++page) {
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/model/ImageConvert.java:52]: Normally there is not a space line after a statement.

File: src/main/java/com/jalasoft/webservice/model/ZipFiles.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+     * Creates a Zipfile with a list of files.
+     *
+     * @param filePaths
+     */
+    public void zipFiles(String [] filePaths) {
+        try {
+            Utils util = new Utils();
+            File firstFile = new File(filePaths[0]);
+            String zipFileName = util.getPublic() + firstFile.getName().concat(".zip");
+
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/model/ZipFiles.java:40]: The line of first must be at the same vertical level of the method.

File: src/main/java/com/jalasoft/webservice/model/ZipFiles.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+            FileOutputStream fos = new FileOutputStream(zipFileName);
+            ZipOutputStream zos = new ZipOutputStream(fos);
+
+            for (String aFile : filePaths) {
+                zos.putNextEntry(new ZipEntry(new File(aFile).getName()));
+
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/model/ZipFiles.java]: There is no reason for having a space.

File: src/main/java/com/jalasoft/webservice/model/ZipFiles.java [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+                byte[] bytes = Files.readAllBytes(Paths.get(aFile));
+                zos.write(bytes, 0, bytes.length);
+                zos.closeEntry();
+            }
+
+            zos.close();
+
+        } catch (FileNotFoundException ex) {
+            System.err.println("A file does not exist: " + ex);
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/model/ZipFiles.java]: In our web service we don´t need to use println.

File: src/main/java/com/jalasoft/webservice/model/ZipFiles.java [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+        } catch (IOException ex) {
+            System.err.println("I/O error: " + ex);
```
Comment (ramalaso) [inline src/main/java/com/jalasoft/webservice/model/ZipFiles.java]: In our web service we don´t need to use println.

**Review by ramalaso — COMMENTED** [review]
Please review the suggestions.

File: src/main/java/com/jalasoft/webservice/model/ImageConvert.java (hunk 2 above)
Comment (FerHinojosa) [reply src/main/java/com/jalasoft/webservice/model/ImageConvert.java]: Solved

File: src/main/java/com/jalasoft/webservice/model/ImageConvert.java (hunk 3 above)
Comment (FerHinojosa) [reply src/main/java/com/jalasoft/webservice/model/ImageConvert.java:52]: solved

File: src/main/java/com/jalasoft/webservice/model/ZipFiles.java (hunk 5 above)
Comment (FerHinojosa) [reply src/main/java/com/jalasoft/webservice/model/ZipFiles.java]: Solved

File: src/main/java/com/jalasoft/webservice/model/ZipFiles.java (hunk 6 above)
Comment (FerHinojosa) [reply src/main/java/com/jalasoft/webservice/model/ZipFiles.java]: Solved

File: src/main/java/com/jalasoft/webservice/model/ZipFiles.java (hunk 7 above)
Comment (FerHinojosa) [reply src/main/java/com/jalasoft/webservice/model/ZipFiles.java]: Solved

File: src/main/java/com/jalasoft/webservice/controller/FileManager.java (hunk 1 above)
Comment (FerHinojosa) [reply src/main/java/com/jalasoft/webservice/controller/FileManager.java]: Solved

**Review by ramalaso — APPROVED** [review]
Ok. Thanks.

**Pull request merged**